api/v1/models/contest.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+const { Model, DataTypes } = require("sequelize");
+
+module.exports = (sequelize) => {
+  class Contest extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
```

Comment:
Model files should not be put under `api/v1`

Reason: They are independent from API version.